src/lib/WireGuard.js

```diff
@@ -94,7 +100,7 @@ module.exports = class WireGuard {
 # Server
 [Interface]
 PrivateKey = ${config.server.privateKey}
-Address = ${config.server.address}/24
+Address = ${config.server.address}/${config.server.cidrBlock}
```

Comment:
The issue is here, would be nice if @willtho89 could get it fixed.

Reply:
What's the issue here? I seem to get the correct CIDR block when I configure.

Reply:
Not an issue anymore thanks to your fixes @utkarsh867